exercises/practice/twelve-days/.meta/example.cpp

```diff
@@ -0,0 +1,51 @@
+#include "twelve_days.h"
+#include <array>
+#include <string>
+
+namespace twelve_days {
+
+std::string verse(int day) {
+    const std::array<std::string, 13> ordinals = {
+        "", "first", "second", "third", "fourth", "fifth", "sixth",
+        "seventh", "eighth", "ninth", "tenth", "eleventh", "twelfth"
+    };
+    
+    const std::array<std::string, 13> gifts = {
+        "", "a Partridge in a Pear Tree", "two Turtle Doves", "three French Hens",
+        "four Calling Birds", "five Gold Rings", "six Geese-a-Laying",
+        "seven Swans-a-Swimming", "eight Maids-a-Milking", "nine Ladies Dancing",
+        "ten Lords-a-Leaping", "eleven Pipers Piping", "twelve Drummers Drumming"
+    };
+    
+    auto result = std::string{"On the "} + ordinals[day] + " day of Christmas my true love gave to me: ";
```

Comment:
This looks like you want to follow my previous suggestion. Then do it properly please.